vacuum-advisor/bloat.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'VACUUM Advisor: Bloat in Postgres'
+backlink_href: /docs/vacuum-advisor
+backlink_title: 'VACUUM Advisor'
+---
+
+In Postgres, bloat refers to a situation that the physical size of a table on
+disk is larger than its actual data size.
+
+## Why does bloat happen?
+
+In Postgres, any DELETEs or UPDATEs create dead rows in the tables they modify.
```

Comment:
I think we should mention MVCC somehow. We don't need to explain it at length, but if you have no idea what that is or why INSERTs or UPDATEs would create dead rows, we can at least give you something to look up. Something like
```suggestion
In Postgres, any DELETEs or UPDATEs create dead rows in the tables they modify. This is due to the MVCC mechanism that Postgres uses to support concurrent transactions.
```